### ISSUE TITLE
SpreadsheetDocument Class documentation contains non-existent Close method

### ISSUE THREAD

**Issue by james-d-hasselman** [dotnet/Open-XML-SDK#1683]
**Describe the bug**
The documentation for the SpreadsheetDocument Class contains a Close method from 2.X which was removed in 3.X. Documentation for the final version of 2.X states that the method will be removed in later releases. Both the 3.0.0 and 3.0.1 documentation need to be updated.

**To Reproduce**
Go [here](https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.packaging.spreadsheetdocument?view=openxml-3.0.1) and search under methods for Close.

**Observed behavior**
I get the build error `'SpreadsheetDocument' does not contain a definition for Close`

**Expected behavior**
The documentation should be updated to reflect the removal of the Close method in release 3.0.0.

**Desktop (please complete the following information):**
 - DocumentFormat.OpenXml Version: 3.0.1

**Comment by thirstyape** [comment]
Agreed, as [this issue](https://github.com/dotnet/Open-XML-SDK/issues/1298) was merged the documentation should reflect that.

**Comment by twsouthwick** [comment]
> Both the 3.0.0 and 3.0.1 documentation need to be updated.

What are envisioning here? The API is not listed as available for v3+ and a large banner is on 2.20.0 saying it's obsolete: https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.packaging.openxmlpackage.close

**Comment by thirstyape** [comment]
I think this may apply to classes that inherit `OpenXmlPackage` such as `SpreadsheetDocument`. You can [see here](https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.packaging.spreadsheetdocument?view=openxml-3.0.1) that the documentation in the 3.0.1 version does not note **Obsolete** as it does in the base class (at least on the summary page, clicking further in does in fact have the correct details).

I would think either having the Obsolete note on the summary page or removing that method from the versions it does not apply to would be ideal.

**Comment by mikeebowen** [comment]
The documentation has been updated to reflect that `.Close()` has been deprecated
